src/software/thunderscope/robot_diagnostics/chicker_widget.py

```diff
@@ -216,17 +206,22 @@ def change_button_state(self, button: QPushButton, enable: bool) -> None:
         """
         if enable:
             button.setStyleSheet("background-color: White")
-            button.setCheckable(True)
+            button.setEnabled(True)
         else:
             button.setStyleSheet("background-color: Grey")
-            button.setCheckable(False)
+            button.setEnabled(False)
```

Comment:
```suggestion
        button.setEnabled(enable)
```

Reply:
consider calling `button.setEnabled(self.kick_chip_buttons_enable)` once instead of twice